proto/pdpb.proto

```diff
@@ -530,10 +532,14 @@ message ScatterRegionRequest {
     // If group is defined, the regions with the same group would be scattered as a whole group.
     // If not defined, the regions would be scattered in a cluster level.
     string group = 5;
+
+    repeated uint64 regions_id = 6;
+    uint64 retry_limit = 7;
 }
 
 message ScatterRegionResponse {
     ResponseHeader header = 1;
+    uint64 finishedPercentage = 2;
```

Comment:
Illegal format

Reply:
This is not a stream request, please add a comment on how to notify multiple times when introducing `percentage`.